packages/new-deploy/src/DeployContext.ts

```diff
@@ -155,32 +221,29 @@ export default class DeployContext extends MultiProvider<config.Domain> {
     this.addCore(domain.name, complete);
   }
 
-  // post-deployment setup
-  // We choose to enroll all currently known routers and watchers here,
-  // event those that may not be immediately connected, so that IF they are
-  // connected in the future, the routers will be properly configured.
-  // Essentially we run likely future governance actions at deploy time, so
-  // we will not be required to run them later.
-  protected async enrollCores(): Promise<ethers.PopulatedTransaction[]> {
```

Comment:
why are some fns in the deploy process protected and some lower down in the call stack aren't?